src/main/java/core/basesyntax/service/handler/impl/SupplyOperationHandler.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.handler.impl;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+import java.util.Map;
+
+public class SupplyOperationHandler implements OperationHandler {
+    private static final int MIN_AMOUNT = 0;
+    private StorageDao storageDao = new StorageDaoImpl();
+
+    @Override
+    public void updateStorage(FruitTransaction transaction) {
+        if (transaction.getAmount() < MIN_AMOUNT) {
```

Comment:
It looks like you are doing this check in several places. Probably we can extract it to some helper utility method?